.github/workflows/ruby-dataset-measure.yml

```diff
@@ -17,6 +17,10 @@ on:
       - .github/workflows/ruby-dataset-measure.yml
   workflow_dispatch:
 
+permissions:
+  contents: read
+  security-events: read
```

Comment:
?

```suggestion
```

Reply:
Indeed, it is not necessary: https://github.com/check-spelling-sandbox/codeql/actions/runs/7824516163/job/21347184556